app/models/chargeHistory/AdjustmentHistoryModel.scala

```diff
@@ -24,5 +24,5 @@ case class AdjustmentHistoryModel(creationEvent: AdjustmentModel,
                                   adjustments: List[AdjustmentModel])
 
 case class AdjustmentModel(amount: BigDecimal,
-                      adjustmentDate: Option[LocalDate],
-                           reasonCode: ReversalReason)
+                           adjustmentDate: Option[LocalDate],
+                           reasonCode: String)
```

Comment:
if we can keep reasonCode to be of type ReversalReason, as in this case we can rely on the compile to spot obvious mistakes in the future, ie if we would have to apply refactoring etc